src/main/java/mate/jdbc/model/Manufacturer.java

```diff
@@ -0,0 +1,45 @@
+package mate.jdbc.model;
+
+public class Manufacturer {
+    private Long id;
+    private String name;
+    private String country;
+
+    public Manufacturer() {
```

Comment:
Implementing a toString method would be beneficial for debugging purposes. It allows you to easily print out a Manufacturer's state, which is helpful during development and for logging.